BraveUI/Design System/Colors/Colors.swift

```diff
@@ -101,9 +101,6 @@ extension UIColor {
   public static var braveOnboarding: UIColor {
     DesignSystemColor.text04.color
   }
```

Comment:
This is still not a design system colour, can we move it out?

Reply:
Fixed at https://github.com/brave/brave-ios/pull/5302/commits/7477d942dc31f61bc6502601ff23a5b7b79c532f